content/blog/ces-reloaded/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Cloud Engineering Summit Reloaded"
+date: 2020-10-16
+meta_desc: "Watch popular talks and panels from the Cloud Engineering Summit."
+meta_image: cloud_engineering.png
+authors:
+    - sophia-parafina
+tags:
+    - cloud engineering summit=
+---
+
+Mission accomplished. Putting on a new event is always risky, especially when the calendar is saturated with virtual conferences. The goal of the Cloud Engineering Summit was to bring the level of conversation above specific technologies and take a holistic look into the future of the cloud and you knocked it out of the park! Here are the stats:
+
+- 46 industry leaders
+- 17 hours of content
+- 3000+ participants
+- 96 countries
+
+<!--more-->
+
+[Joe Duffy's](https://twitter.com/funcofjoe) keynote kicked off the summit by describing the current state of cloud engineer and what the future holds for cloud engineering. Watch Joe's keynote, [The Future of Cloud Engineering](https://cloudengineering.heysummit.com/talks/welcome-keynote/) to set the stage for the summit.
```

Comment:
there's something wrong with this sentence.

in addition, i think we should be consistent in Cloud Engineer(ing) casing ... I favor "promoting" it (reasonable folks can differ), however, we should pick a lane :)